content/projects/redux-with-angular/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Redux with Angular
+ready: true
+prerequisites:
+  hard: ["topics/redux-with-angular"]
```

Comment:
we do have some redux material in the repo. It's probably in the wrong place, but worth referring to. 
If it is sufficient it would be good to use it. If it is not sufficient then it would be good to upgrade it.

Reply:
ngrx which is the implementation of redux is a completely new implementation relative to angular. I agree that it can be done but it would have time to merge the two concepts and explain the difference. I don't have time to focus on it now. I have a lot of content to cover.

Reply:
With React+redux stuff I found it useful to show people how to use redux in isolation before teaching react + redux. It's just easier to understand that way.  Redux is actually pretty easy to understand on its own. 

Looking at the tutorial you linked to, it doesn't explain redux very well. I just looked at the text though. I mean "Actions are plain JS objects that represent something that has happened. Can be compared to events." I can't imagine being satisfied with that definition. It's not a useful definition. Maybe the videos go into more detail but I'm not so certain. 

An action is a plian ol json object. It can hold more or less anything you can turn into a string. It has to have a `type` key. But still, this explanation is insufficient.

The existing content is for js, not ts. But the foundational concepts are the same. I've seen quite a few recruits suffer over redux. But it doesn't need to be weird. Like, all the recruits really. 

I'd really strongly recommend getting people to understand redux in isolation. Link to the existing stuff, if you want I can come up with a pure redux project. But I havent used ts in years so we could tag team it if you want. I can make the project with js syntax, you can add the ts stuff.

Reply:
I really do not have time to do all this now. I am pushing for Mobile content to be done so i can focus on project submissions. If the content is good enough for them to move on I need it to be done so i can move on. I have over 10 projects to create and upload this week.